puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/packaging_build_deb_coreproject.yaml

```diff
@@ -29,7 +29,7 @@
           values:
           - debian
     execution-strategy:
-      combination-filter: '((onlyarch == "all") || onlyarch == arch) && ((onlyos == "all") || onlyos == os) && !((version != "1.15" || (arch == "armv7" || arch == "armv8")) && os == "trusty") && !((version != "1.15" && version != "1.16") && os == "jessie") && !(version == "1.15" && os == "stretch") && !((version != "1.15" || version != "1.16" || version != 1.17) && arch == "armv7")'
+      combination-filter: '((onlyarch == "all") || onlyarch == arch) && ((onlyos == "all") || onlyos == os) && !((arch == "armv7" || arch == "armv8") && os == "trusty") && !(version != "1.16" && os == "jessie") && !((version != "1.16" || version != 1.17 || version != "1.18") && arch == "armv7")'
```

Comment:
in the last condition, 1.18 should not get added, see GH-654

Reply:
1.15 was the last release to support Ubuntu/trusty, so it can get deleted here (and in the other jobs), too. But I would also be fine if that's left to another PR.